warehouse/ingest-core/src/main/java/datawave/ingest/util/GenerateMultipleNumShardsCacheFile.java

```diff
@@ -24,6 +24,7 @@
     public static final String CONFIG_DIRECTORY_LOCATION_OVERRIDE = "cd";
     public static final String CONFIG_SUFFIEX_OVERRIDE = "cs";
     
+    @SuppressWarnings("static-access")
```

Comment:
This caused me to look at why you were adding this and didn't fix the underlying static access problem. This code is dangerous. It appears OptionBuilder sets static variables. I think we're safe since we're always setting the same flags on the option, but if we weren't, we'd have a bug since we're not calling OptionBuilder.reset between each use. Not suggesting you need to change anything for this PR, but just noticed that.